src/Mealz/MealBundle/Resources/views/Slot/list.html.twig

```diff
@@ -0,0 +1,67 @@
+{% extends 'base_navigation.html.twig' %}
+
+{% import 'MealzTemplateBundle::Components/button.html.twig' as MealzButton %}
+{% import 'MealzTemplateBundle::Components/checkbox.html.twig' as MealzCheckbox %}
+{% import 'MealzTemplateBundle::Components/headline.html.twig' as MealzHeadline %}
+{% import 'MealzTemplateBundle::Components/table.html.twig' as MealzTable %}
+
+{% block pageView %}slotIndex{% endblock %}
+{% set wrapperClass = 'slot-list form-ajax' %}
+
+{% block body %}
+{# Set data for table header #}
+
+{% set tableHeader = [{
+        'label': 'table.title'|trans({}, 'general'),
+        'class': 'slot-title'
+    }, {
+        'label': 'Limit',
+        'class': 'slot-limit'
+    }, {
+        'label': 'table.actions'|trans({}, 'general'),
+        'class': 'slot-actions no-sort'
+    }]
+%}
+
+{# Set data for table content #}
+
+{% set tableContent = [] %}
+{% for slot in entities %}
+    {% set tableContent = tableContent | merge([{
+        'title': {
+            'content': slot.title,
+            'class': 'slot-title'
+        },
+        'limit': {
+            'content': slot.limit > 0 ? slot.limit : '',
+            'class': 'slot-limit'
+        },
+        'actions': {
+            'content': MealzButton.buttonbar('', 'buttonbar-slot-actions', [
+                MealzButton.link(path('MealzMealBundle_Slot_edit', {'slug': slot.slug}), 'button.edit'|trans({}, 'actions'), '', 'button-table load-ajax-form load-edit-form transparent', 'glyphicon-pencil'),
+                '<input type="checkbox" id="slot'~slot.id~'" class="js-switch" name="" style="display: block;" value="1" '~ (slot.enabled ? 'checked' : '') ~' data-id="'~slot.slug~'" />',
```

Comment:
Können die Switchery Checkbox später in ein eigenes Template (analog zur Buttonbar, Buttons, Headline, ...) packen. Habe sowas ähnliches für meine Story umgesetzt.

Reply:
Das gucke es mir dann an.

Reply:
Ich habe es jetzt geändert. Anstatt HTML direkt zu verwenden, verwende ich jetzt ein Macro.

Reply:
Cool 👍